app/templates/components/site-head.hbs

```diff
@@ -19,8 +19,11 @@
   }}
 </div>
 {{wds-banner-notifications}}
-{{wds-community-bar
-  model=communityHeader
-  track=(action track)
-  onLinkClicked=(action onLinkClicked)
-}}
+{{! It's not available on language wikis index }}
+{{#if communityHeader}}
+  {{wds-community-bar
```

Comment:
while at it would you mind:
```suggestion
  <WdsCommunityBar @model={{communityHeader}} @track={{action track}} onLinkClicked={{action onLinkClicked}} />
```

Reply:
I see that we don't use angle brackets anywhere in mobile-wiki yet. I'd prefer to not introduce them in this PR as it can have unknown side-effects and I don't want to leave the Platform team with a problem like this to fix. I suggest converting them all in a single, separate PR, instead.